homeassistant/components/camera/mqtt.py

```diff
@@ -12,30 +12,34 @@
 
 from homeassistant.core import callback
 import homeassistant.components.mqtt as mqtt
+from homeassistant.components.mqtt import CONF_STATE_TOPIC
```

Comment:
Let's not do this as it doesn't make sense. The MQTT image is not going to be the state of the entity.